README.md

```diff
@@ -1,22 +1,68 @@
+| --------- | ---------- |
+| Raid | A Raid is the OSS project's issue that we are working to resolve - see how we [converted react-hooks-testing-library to TypeScript](https://github.com/testing-library/react-hooks-testing-library/pull/520) for an example |
+| Raid Boss | The Raid Boss is the issue that we are 'fighting' to resolve |
+| Raid Group | The collective noun for all contributors to the Raid |
+| Raid Repo | The repository for the current Raid |
+
+## Storyline Quest (mission) :scroll:
+
+Objectives:
+
+* To make contributing to OSS enjoyable, engaging, and with a low barrier-to-entry for those who have never done it before
+* To gamify the process of mob programming OSS issues
+* To have a high impact in a small amount of time; specifically targetting low-medium contributor activity, medium-hgih usage of the project with only a handful of maintainers
+
+<!-- Previous points
 - Themed from MMORPG gaming, gamifiying Mob programming OSS repos
 - Look for criteria in OSS repos 
   1. Low to medium contributor activity, only a handful of maintainers
   2. Medium to high usage of the OSS 
   3. Our impact can be potentially high; in a few days
+-->
+
+## Join Us! :smiling_face_with_three_hearts:
+
+Anoyone is welcome to participate in our Raids - no matter your experience level! Never contributed to an OSS project? No worries! We are all learning something or another, so come along for the ride!
+
+### Intake Quests
+
+1. Say hello to your fellow raiders in the #:bee:-raid-general channel on [Discord][discord]
+2. Raise your first issue against a raid, and start contributing!
+
+### Advancements
+
+When you are:
+* Actively contributing to Raids
+* Being an active community member on the [KCD Discord][discord]
+
+Then we can determine responsibilities and get you running as either an Org member or Collab (reduced permissions)
+
+## Raids :crossed_swords:
+
+* One Raid at a time to maximize impact in a short amount of time.
+
+### Communication :phone:
+
+We primarily communicate via [Discord][discord] in the `OSS-RAID-GROUP` category
+
+### Workflow :gear:
 
-## Joining KCD Raid Group :smiling_face_with_three_hearts:
-1. Active Raid contributor
-2. Active Community member (KCD) 
-3. Determine responsibilities then set to either Org member or Collab (reduced permissions)
+When participating in a Raid, use the following steps.
 
-## The Process find target -> communicate -> Raid -> Push Code :fire:
-1. One Target at a time to maximize impact in short amount of time.
+1. Raise an issue in the Raid repo with a one-liner title describing what you are working on - for example `Migrate page-header/breadcrumbs tests to testing-library`
+2. Fork the Raid Repo
+3. Create a branch named after your GitHub username - e.g. `nobrayner`
+4. Hack away :sunglasses::computer:
+5. Create a PR titled the same as your issue, and a desciption of `Fixes #{ISSUE_NO}` - [like this PR for example](https://github.com/kcdraidgroup/hospitalrun-frontend/pull/8)
```

Comment:
@JacobMGEvans This is the process I have gleaned from the chat. I'd say it's good enough for now?

Reply:
I'm not sure why you need a branch with your username if you're already working in a fork?  If anything I'd want to branch for my issue number so I can work on the next issue while another is up for review.

Reply:
> I'm not sure why you need a branch with your username if you're already working in a fork? If anything I'd want to branch for my issue number so I can work on the next issue while another is up for review.

The issues are really your normal ticket, they are just there to track who is working on what, ultimately the PRs are just mild protection of the forked main/master but still hopefully keep the move fast capability.